csharp/src/Ice/LocatorInfo.cs

```diff
@@ -101,11 +104,11 @@ internal async ValueTask<(IReadOnlyList<Endpoint>, bool)> GetEndpointsAsync(Refe
                         // Reference is returned from the cache but TTL was reached, if backgrounds updates
                         // are configured, we obtain a new reference to refresh the cache but use the stale
                         // reference to not block the caller.
-                        _ = GetObjectReferenceAsync(reference);
+                        _ = GetObjectReferenceAsync(reference, cancel: default);
```

Comment:
It's not clear to me why you don't pass the `cancel` parameter here.

Reply:
The background feature is supposed to allow refreshing the endpoints from time to time, when TTL expired. I don't see the point of canceling this background refresh if the request that triggered it is canceled.